cognite/neat/_session/_inspect.py

```diff
@@ -8,8 +14,76 @@
 class InspectAPI:
     def __init__(self, state: SessionState) -> None:
         self._state = state
+        self.issues = InspectIssues(state)
 
     @property
     def properties(self) -> pd.DataFrame:
         """Returns the properties of the current data model."""
         return self._state.last_verified_rule.properties.to_pandas()
+
+
+@intercept_session_exceptions
+class InspectIssues:
+    """Inspect issues of the current data model."""
+
+    def __init__(self, state: SessionState) -> None:
+        self._state = state
+
+    @overload
+    def __call__(
+        self,
+        search: str | None = None,
+        return_dataframe: Literal[True] = (False if IN_NOTEBOOK else True),  # type: ignore[assignment]
+    ) -> pd.DataFrame: ...
+
+    @overload
+    def __call__(
+        self,
+        search: str | None = None,
+        return_dataframe: Literal[False] = (False if IN_NOTEBOOK else True),  # type: ignore[assignment]
+    ) -> None: ...
+
+    def __call__(
+        self,
+        search: str | None = None,
+        return_dataframe: bool = (False if IN_NOTEBOOK else True),  # type: ignore[assignment]
+    ) -> pd.DataFrame | None:
+        """Returns the issues of the current data model."""
+        issues = self._state.last_issues
+        if not issues:
+            self._print("No issues found.")
+
+        if issues and search is not None:
+            unique_types = {type(issue).__name__ for issue in issues}
+            closest_match = set(difflib.get_close_matches(search, unique_types))
```

Comment:
will this return partial matches ? what about fuzzy instead ? https://github.com/seatgeek/thefuzz or using cognite sdk entity matching ?

Reply:
This is a very simple way of using the standard library without any extra dependencies to get a slightly better search experience. Suggest we avoid any extra dependency